src/graphql/schema.js

```diff
@@ -21,11 +23,14 @@ const Spell = require('../models/spell');
 const StartingEquipment = require('../models/startingEquipment');
```

Comment:
You'll also need to update the models for `StartingEquipment` and `Features` since we're adding the `background` field to both.